ddl/ddl_db_change_test.go

```diff
@@ -292,14 +292,38 @@ type sqlWithErr struct {
 	expectErr error
 }
 
+type expectQuery struct {
+	sql  string
+	rows []string
+}
+
+// https://github.com/pingcap/tidb/pull/6249 fixes the following two test cases.
+func (s *testStateChangeSuite) TestWriteOnlyWriteNULL(c *C) {
+	sqls := make([]sqlWithErr, 1)
+	sqls[0] = sqlWithErr{"insert t set c1 = 'c1_new', c3 = '2019-02-12', c4 = 8 on duplicate key update c1 = values(c1)", nil}
+	addColumnSQL := "alter table t add column c5 int not null default 1 after c4"
+	expectQuery := &expectQuery{"select c4, c5 from t", []string{"8 1"}}
+	s.runTestInSchemaState(c, model.StateWriteReorganization, "", addColumnSQL, sqls, expectQuery)
+}
+
+func (s *testStateChangeSuite) TestWriteOnlyOnDupUpdate(c *C) {
```

Comment:
s/WriteOnly/WriteReorg ? I only want to unify these test names.
I don't understand why this function's name and the next function's name both use "TestWriteOnly". But they use the different schema state.

Reply:
In my opinion, test cases should first test what they want to test, and secondly, their names should match what they want to test. Whether the code used in its case needs to follow the use case name is not always necessary.

Reply:
But your SQL are all executed in the write-reorg state. Why did you say it tests the write-only state?

Reply:
But it should fail in write only state.

Reply:
I've added the comments for it.